README.md

```diff
@@ -1971,93 +1970,93 @@ includes equity compensation, or to evaluate equity you currently have in a comp
-equity. In later stage private companies, the financials of the company are better
-understood (at least to investors and others with an inside view of the company), and
-these predictions are often more certain.
+In startups there tends to be a high degree of uncertainty about the future value of
+equity, while in later stage private companies financials are better understood (at least
+to investors and others with an inside view of the company), and these predictions are
+often more certain.
 
 ### Dangers evaluating equity compensation
 
-Some of these items have already been discussed, but all are important enough to list
-together:
+Some of these items have already been discussed, but they’re important enough to consider
+all together when trying to assign value to your equity:
 
 - ❗ **Details matter**: When it comes to equity compensation, details matter!
-  You need to understand the type of stock grant or stock option in detail, and what it
-  means for your taxes, to know the likely value.
+  You need to understand the type of stock grant or stock option in detail, as well as what
+  it means for your taxes, to know what your equity is worth.
 - ❗ **Seek professional advice and understand it**: Because details are so important,
-  [professional advice](#seeking-professional-advice) from a tax advisor and a lawyer familiar
-  with equity compensation is often a good idea.
+  [professional advice](#seeking-professional-advice) from a tax advisor or lawyer familiar with
+  equity compensation (or both) is often a good idea.
   Avoid doing everything yourself, but also avoid blindly trusting advisors without having
   them explain the details to you in a way you understand.
 - ❗ In some cases, high taxes may prevent you from exercising your options, and if you can’t
-  sell the stock, you could effectively be forced to walk away from it if you can’t afford
-  to exercise.
+  sell the stock, you could effectively be forced to walk away.
 - ❗ Once you get an offer, you need to understand the value of the equity component.
   You need quite a bit of information to figure this out, and should just ask.
   If the company trusts you enough to be making an offer, and still doesn’t want to answer
-  these questions about that offer, it’s **a warning sign**. See
-  [our list](#questions-to-ask-about-offers).
+  questions about that offer, consider it **a warning sign**. The section that follows lists
+  the questions a company should reasonably be expected to answer about offers they make.
 
 ### Questions to ask about offers
 
-🔹 It’s helpful to ask questions after you get an offer.
-Here are a few especially important ones especially relevant to startups and other private
-companies:
+🔹 It’s important to ask questions when you get an offer that includes any kind of equity.
+In addition to learning what you need to know to evaluate your offer, this process can you
+help you evaluate the company making the offer as well.
```

Comment:
Tightening: elide "as well"

Reply:
Or perhaps clearer:

In addition to helping you learn the facts about the equity offer, the process of discussing these details can help you get a sense for the company's responsiveness and transparency.

Reply:
Fixed. (Replaced with suggested text.)